.conda/env_build.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+    - accessnri
+    - conda-forge
+    - coecms
```

Comment:
Out of curiosity, when do we get rid of the `coecms` dependency, given it's shut down now?

Reply:
There's a dependency on `nchash` (https://github.com/aidanheerdegen/nchash) which is deployed to the `coecms` conda channel. Though I'm unsure on whether the dependency is still required - see this existing issue: https://github.com/ACCESS-NRI/yamanifest/issues/11.